mofacts/client/lib/router.js

```diff
@@ -247,7 +246,7 @@ Router.route('/profile', {
       if (loginMode === 'southwest') {
         console.log('southwest login, routing to southwest profile');
         Session.set('curModule', 'profileSouthwest');
-        this.redirect('/profileSouthwest');
```

Comment:
redirecting to the route will cause iron router to not render the profile page.

Reply:
this isn't true, tested it with live data.

Reply:
Also, did you mean to comment on the existing code?

Reply:
meant to comment on the new code

Reply:
please test again with the signinsouthwest route.

Reply:
worked fine still, but I will change it to a render.